build.gradle

```diff
@@ -42,7 +42,11 @@ ext {
 
 }
 
-mainClassName = 'pcgen.system.Main'
+//The gradle plugin for netbeans sets the 'mainClass' property instead of 'mainClassName'
```

Comment:
I actually have no idea, but I can't find a config for it and my change doesn't affect the default behavior outside of my IDE.